bluemira/builders/EUDEMO/tools.py

```diff
@@ -280,6 +281,26 @@ def _order_shapes_anticlockwise(shapes):
     return list(np.array(shapes)[indices])
 
 
+def find_xy_plane_radii(wire, plane):
+    """
+    Get the radial coordinates of a wire's intersection points with a plane.
+
+    Parameters
+    ----------
+    wire: BluemiraWire
+        Wire to get the radii for in the plane
+    plane: BluemiraPlacement
+        Plane to slice with
+
+    Returns
+    -------
+    radii: np.ndarray
+        The array of radii of intersections, sorted from smallest to largest
+    """
+    intersections = slice_shape(wire, plane)
+    return sorted(intersections[:, 0])
```

Comment:
Won't `sorted()` return a `list`? The docstring says `ndarray`; maybe we want `np.sort` instead?

Reply:
Actually, I'm a tiny bit confused with the usage of `slice_shape` here and the function's documentation.

The docstring indicates that `slice_shape` returns one of:

- `Tuple[List[ndarray], List[BluemiraWire]]`
- `Tuple[List[ndarray], None]`
- `Tuple[None, List[BluemiraWire]]`
- `Tuple[None, None]`

This function seems to be treating the return value as an `ndarray`. Is the documentation wrong?